### PR TITLE
fix: listening 0.0.0.0 when use docker run greptimedb

### PR DESCRIPTION
**Pull request by zyy17**

### CODE REVIEW

**Review by xtang — APPROVED** [review]
LGTM.
@sunng87 PTAL

File: docs/installation/download-pre-built.md [hunk 1]
```diff
@@ -26,7 +26,13 @@ Make sure the [Docker](https://www.docker.com/) is already installed. If not, yo
 docker run -p 4000-4004:4000-4004 \
 -p 4242:4242 -v "greptime-vol:/tmp/greptimedb" \
 --name greptime --rm \
-greptime/greptimedb standalone start
+greptime/greptimedb standalone start \
+--http-addr 0.0.0.0:4000 \
+--rpc-addr 0.0.0.0:4001 \
```
Comment (sunng87) [inline docs/installation/download-pre-built.md:31]:
```suggestion
--grpc-addr 0.0.0.0:4001 \
```

File: docs/installation/download-pre-built.md [hunk 2]
```diff
@@ -43,7 +49,13 @@ You can:
    docker run --security-opt seccomp=unconfined -p 4000-4004:4000-4004 \
    -p 4242:4242 -v "greptime-vol:/tmp/greptimedb" \
    --name greptime --rm \
-   greptime/greptimedb standalone start
+   greptime/greptimedb standalone start \
+   --http-addr 0.0.0.0:4000 \
+   --rpc-addr 0.0.0.0:4001 \
```
Comment (sunng87) [inline docs/installation/download-pre-built.md:54]:
```suggestion
   --grpc-addr 0.0.0.0:4001 \
```

File: docs/installation/download-pre-built.md [hunk 3]
```diff
@@ -26,7 +26,13 @@ Make sure the [Docker](https://www.docker.com/) is already installed. If not, yo
 docker run -p 4000-4004:4000-4004 \
 -p 4242:4242 -v "greptime-vol:/tmp/greptimedb" \
 --name greptime --rm \
-greptime/greptimedb standalone start
+greptime/greptimedb standalone start \
+--http-addr 0.0.0.0:4000 \
+--grpc-addr 0.0.0.0:4001 \
```
Comment (sunng87) [inline docs/installation/download-pre-built.md:31]:
```suggestion
--rpc-addr 0.0.0.0:4001 \
```

File: docs/installation/download-pre-built.md [hunk 4]
```diff
@@ -43,7 +49,13 @@ You can:
    docker run --security-opt seccomp=unconfined -p 4000-4004:4000-4004 \
    -p 4242:4242 -v "greptime-vol:/tmp/greptimedb" \
    --name greptime --rm \
-   greptime/greptimedb standalone start
+   greptime/greptimedb standalone start \
+   --http-addr 0.0.0.0:4000 \
+   --grpc-addr 0.0.0.0:4001 \
```
Comment (sunng87) [inline docs/installation/download-pre-built.md:54]:
```suggestion
   --rpc-addr 0.0.0.0:4001 \
```

**Review by waynexia — APPROVED** [review]

**Review by waynexia — CHANGES_REQUESTED** [review]
wait a second...

**Review by sunng87 — APPROVED** [review]

**Comment by sunng87** [comment]
@waynexia I just reverted my change. It should be fine now

**Review by waynexia — APPROVED** [review]

**Pull request merged**